app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsFragment.kt

```diff
@@ -106,10 +106,7 @@ class SuggestedEditsCardsFragment : Fragment(), MenuProvider, SuggestedEditsItem
         super.onViewCreated(view, savedInstanceState)
         requireActivity().addMenuProvider(this, viewLifecycleOwner, Lifecycle.State.RESUMED)
         setInitialUiState()
-        // Disable offscreenPageLimit for recentEdits can avoid the tooltips showing incorrectly.
-        if (action != VANDALISM_PATROL) {
-            binding.cardsViewPager.offscreenPageLimit = 2
-        }
```

Comment:
Hi @dbrant 

The reason why I put the logic here was trying to avoid the following issue:
<img src="https://github.com/wikimedia/apps-android-wikipedia/assets/2435576/71ee9653-8347-4617-b68f-50858a30b7d1" width="400">

Reply:
ok, so let's fix the underlying issue. Done!